### PR TITLE
Added integration test scripts

### PR DESCRIPTION
**Pull request by jshaofuturewei**

### CODE REVIEW

File: hack/globalscheduler/test/create_clusters.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    app: test
+spec:
+  ipaddress: "34.218.224.247"
+  geolocation:
+    city: "BBB"
+    province: "King"
+    area: "Downtown"
+    country: "USA"
+  region:
+    region: "NW"
```
Comment (kimeunju108) [inline hack/globalscheduler/test/create_clusters.sh:34]: If region is other value such as "east-1", does it affect dispatcher or scheduler?

File: hack/globalscheduler/test/create_clusters.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  storage:
+    - typeid: "ssd"
+      storagecapacity: 12
+    - typeid: "sas"
+      storagecapacity: 2200
+  eipcapacity: 100
+  cpucapacity: 200
+  memcapacity: 300
+  serverprice: 400
+  homescheduler: "mockscheduler"
```
Comment (kimeunju108) [inline hack/globalscheduler/test/create_clusters.sh:52]: If homescheduler has other default value such as "scheduler0", does it affect distributor, dispatcher or scheduler?

File: hack/globalscheduler/test/create_clusters.sh (hunk 1 above)
Comment (jshaofuturewei) [reply hack/globalscheduler/test/create_clusters.sh:34]: Region does not have any impacts.

File: hack/globalscheduler/test/create_clusters.sh (hunk 2 above)
Comment (jshaofuturewei) [reply hack/globalscheduler/test/create_clusters.sh:52]: It shall be updated by scheduler.

**Review by CoderKevinZhang — APPROVED** [review]

**Review by kimeunju108 — APPROVED** [review]
LGTM

**Review by cathyhongzhang — APPROVED** [review]
LGTM

**Pull request merged**